.github/workflows/ci.yml

```diff
@@ -214,7 +214,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
         with:
-          path: ${{ matrix.os == 'macos-latest' && 'src-tauri/target/universal-apple-darwin/release/bundle/*/*' || 'src-tauri/target/release/bundle/*/*' }}
+          path: ${{ matrix.os == 'macos-latest' && 'src-tauri/target/universal-apple-darwin/*/bundle/*/*' || 'src-tauri/target/*/bundle/*/*' }}
```

Comment:
Actually this will also break the paths in `publish-apps-release` @adamchalmers, if you look in the zip at `https://github.com/KittyCAD/modeling-app/actions/runs/6775340739?pr=1005`, there's now a top-level `debug/bundle/` inside the artifact.zip

Reply:
What I'm saying is if we merge as-is the release CI will fail

Reply:
It's really not abstracted out in the `publish-apps-release` section

Reply:
I guess we need a var here that's either debug (in which case no `publish-apps-release`) or `release`. Let me push that

Reply:
Done, let's see